src/tests/unit/packages/bpmn/bpmn-group/bpmn-group-component-test.tsx

```diff
@@ -0,0 +1,22 @@
+import { wrappedRender } from '../../../test-utils/render';
+import * as React from 'react';
+import { CSSProperties } from 'react';
+import { Multiline } from '../../../../../main/utils/svg/multiline';
+import { BPMNGroup } from '../../../../../main/packages/bpmn/bpmn-group/bpmn-group';
+import { BPMNGroupComponent } from '../../../../../main/packages/bpmn/bpmn-group/bpmn-group-component';
+
+// override getStringWidth, because it uses by jsdom unsupported SVGElement methods
```

Comment:
@matthiaslehnertum I don't need this patching for getting the test to pass. As with the other PR, perhaps this is a JSDOM versioning issue? I test using a fresh clone so perhaps I'm getting the latest compatible version.

Reply:
@loreanvictor Thank you for the in-depth testing! :) I managed to get the disassociation issue fix. The SVG method patch also does not seem to be necessary still - so I removed that as well.